### PR TITLE
new Units to UCUM

### PR DESCRIPTION
**Pull request by irinazherko**

### CODE REVIEW

**Comment by hardhouse** [comment]
@irinazherko pConcept_code =>'kJ/mol' - twice

File: working/manual_changes/2022/2022-02-28.sql [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+pConcept_class_id =>'Unit',
+pStandard_concept =>'S',
+pConcept_code =>'mbar/L'
+);
+END $_$;
+
+DO $_$
+BEGIN
+PERFORM vocabulary_pack.AddNewConcept(
+pConcept_name =>'fraction',
```
Comment (TinyRickC137) [inline working/manual_changes/2022/2022-02-28.sql:364]: Unclear, can we use ratio here?

Comment (TinyRickC137) [reply working/manual_changes/2022/2022-02-28.sql:364]: Discussed with Irina - agreement on 'fraction' achieved inside the team before adding this unit

**Comment by TinyRickC137** [comment]
Can we wait for the release until Friday evening (March 25)? We are mapping Snomed units now, hence potential new UCUM concepts coming

**Review by hardhouse — APPROVED** [review]

**Pull request closed without merging**